### PR TITLE
Updating meeting info for July 2019.

### PR DESCRIPTION
**Pull request by djgoku**

### CODE REVIEW

File: lib/kc_elixir_web/controllers/page_controller.ex [hunk 1]
```diff
@@ -4,13 +4,13 @@ defmodule KcElixirWeb.PageController do
...
-        ~N{2019-06-06 20:00:00}
+        ~N{2019-07-11 18:00:00},
+        ~N{2019-07-11 20:00:00}
       ],
       speaker: %{
-        name: "Jeremy Owens-Boggs"
+        name: "Everyone"
       },
-      topic: "Macros - use to document pubsub",
+      topic: "Project Structures - ☂ projects - Directories on my!",
```
Comment (seancribbs) [inline lib/kc_elixir_web/controllers/page_controller.ex]: s/on/oh/

Comment (adkron) [reply lib/kc_elixir_web/controllers/page_controller.ex]: It should have a comma too. Directories, oh my.

I believe.

Comment (adkron) [reply lib/kc_elixir_web/controllers/page_controller.ex]: It should have a comma too. Directories, oh my.

I believe.

Comment (djgoku) [reply lib/kc_elixir_web/controllers/page_controller.ex]: Fixed!

**Pull request merged**